onadata/apps/api/viewsets/data_viewset.py

```diff
@@ -261,12 +261,16 @@ def enketo(self, request, *args, **kwargs):
         elif(isinstance(self.object, Instance)):
             if request.user.has_perm("change_xform", self.object.xform):
                 return_url = request.query_params.get('return_url')
```

Comment:
This variable doesn't seem to be used in this function... Should it have been passed into `get_enketo_urls` ? Seems the previous `get_enketo_edit_url` function generated edit URLs with a return URL ? I also wonder if Enketo or another client will be able to correctly use the data we now return from this endpoint.... Seems the response has now completely changed?

Reply:
<img width="902" alt="Screenshot 2020-08-24 at 15 08 49" src="https://user-images.githubusercontent.com/25849009/91043149-b8f83080-e61b-11ea-9f25-dd8fcae3aa5f.png">

Reply:
> Seems the previous get_enketo_edit_url function generated edit URLs with a return URL ?

From what I gather the `edit_url` is in this case the value of `url` when we hit this endpoint `http -a <enketo-key>: POST :8005/api_v2/survey/all server_url=="https://api.ona.io/<username>" form_id=<id_string>`

So I dont believe anything has broken really in the new implementation. 

> I also wonder if Enketo or another client will be able to correctly use the data we now return from this endpoint

I believe they should be able to use this url. Having this QA'd might be more insightful though.